functional-tests/build.gradle

```diff
@@ -79,6 +79,21 @@ drivers.each { driver ->
 
         systemProperty "geb.build.reportsDir", reporting.file("geb/$name")
         systemProperty "geb.env", driver
+
+        /*
+         * Set environment variables from a .env file, if it exists.
+         * Equals sign separated key value pairs, one per new line: SOME_KEY=some value
+         * These local environment variables take precedence over any environment variables set at the system level.
+         */
+        def envFile = file('.env')
+        if (envFile) {
+            envFile.readLines().each() {
+                def (key, value) = it.tokenize('=')
+                if (key && value) {
+                    environment key, value
```

Comment:
This logic reads a .env file at this level and sets the environment variables.  Should make running the functional tests locally easier.

Reply:
I would prefer it read form the environment itself only, and not a .env file. I just think it may be confusing, you'd have to have read this code, or some documentation to know which takes precedence.
Again - if we're all using docker, then the need for this code would go away.